plugins/inputs/vsphere/README.md

```diff
@@ -551,9 +561,231 @@ For a detailed list of commonly available metrics, please refer to
 * virtualDisk stats for VM
   * disk (name of virtual disk)
 
+## Add a vSAN extension
+
+A vSAN resource is a special type of resource that can be collected by the
+plugin. The configuration of a vSAN resource slightly differs from the
+configuration of hosts, VMs, and other resources.
+
+### Prerequisites for vSAN
+
+* vSphere 6.5 and later
+* Clusters with vSAN enabled
+* [Turn on Virtual SAN performance service](https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.virtualsan.doc/GUID-02F67DC3-3D5A-48A4-A445-D2BD6AF2862C.html): When you create a vSAN cluster,
+the performance service is disabled. To monitor the performance metrics,
+you must turn on vSAN performance service.
+
+### vSAN Configuration
+
+```toml
+[[inputs.vsphere]]
+  interval = "300s"
+  vcenters = ["https://<vcenter-ip>/sdk", "https://<vcenter2-ip>/sdk"]
+  username = "<user>"
+  password = "<pwd>"
+
+  # Exclude all other metrics
+  vm_metric_exclude = ["*"]
+  datastore_metric_exclude = ["*"]
+  datacenter_metric_exclude = ["*"]
+  host_metric_exclude = ["*"]
+  cluster_metric_exclude = ["*"]
+  
+  # By default all supported entity will be included
+  vsan_metric_include = [
+    "summary.disk-usage",
+    "summary.health",
+    "summary.resync",
+    "performance.cluster-domclient",
+    "performance.cluster-domcompmgr",
+    "performance.host-domclient",
+    "performance.host-domcompmgr",
+    "performance.cache-disk",
+    "performance.disk-group",
+    "performance.capacity-disk",
+    "performance.disk-group",
+    "performance.virtual-machine",
+    "performance.vscsi",
+    "performance.virtual-disk",
+    "performance.vsan-host-net",
+    "performance.vsan-vnic-net",
+    "performance.vsan-pnic-net",
+    "performance.vsan-iscsi-host",
+    "performance.vsan-iscsi-target",
+    "performance.vsan-iscsi-lun",
+    "performance.lsom-world-cpu",
+    "performance.nic-world-cpu",
+    "performance.dom-world-cpu",
+    "performance.cmmds-world-cpu",
+    "performance.host-cpu",
+    "performance.host-domowner",
+    "performance.host-memory-slab",
+    "performance.host-memory-heap",
+    "performance.system-mem",
+  ]
+  # by default vsan_metric_skip_verify = false
+  vsan_metric_skip_verify = true
+  # vsan_metric_exclude = [ ]
```

Comment:
I still think this needs to be uncommented, as the default is `*` which should exclude all the defined metrics?
```suggestion
  vsan_metric_exclude = [ ]
```